website/src/pages/styles.module.css

```diff
@@ -125,6 +125,11 @@
   vertical-align: sub;
 }
 
+.indexCtasGitHubButton iframe {
```

Comment:
Can we add an id or a class to the `iframe`?

Reply:
Done :heavy_check_mark: 

BTW, what does "Ctas" mean? Is this by chance not "Stars"?

Reply:
CTA means Call-to-action lol. Things get lost in translation when we use acronyms with bad casing.

Reply:
Oops :face_with_head_bandage:, I would rename it to something else, but this is not an important thing.